src/SprykerEco/Shared/Econda/EcondaConstants.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
 /**
- * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * MIT License
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
 namespace SprykerEco\Shared\Econda;
 
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
 interface EcondaConstants
 {
     public const CSV_FOLDER_PATH = 'ECONDA:ECONDA_CSV_FOLDER_PATH';
     public const API_ID = 'ECONDA:API_ID';
+    public const ACCOUNT_ID = 'ECONDA:ACCOUNT_ID';
```

Comment:
Example in https://github.com/spryker-eco/econda/blob/master/config/config.dist.php is missing.

Reply:
No getter for the config value and example how to pass to the econda component.